pkg/registry/core/node/strategy.go

```diff
@@ -194,13 +194,13 @@ func NodeNameTriggerFunc(obj runtime.Object) []pkgstorage.MatchValue {
 }
 
 // ResourceLocation returns a URL and transport which one can use to send traffic for the specified node.
-func ResourceLocation(getter ResourceGetter, connection client.ConnectionInfoGetter, proxyTransport http.RoundTripper, ctx context.Context, id string) (*url.URL, http.RoundTripper, error) {
```

Comment:
when cleaning context, i happen to find that `ResourceGetter` here is also unused. now it's cleaned. (to see if ci looks happy)